torch/_inductor/select_algorithm.py

```diff
@@ -166,11 +166,25 @@ class PartialRender:
     of replacements after the initial render.
     """
 
+    FINALIZED_HOOK: None = None
```

Comment:
nit: for a sentinel object, would:

`FINALIZED_HOOK: object = object()` be safer ?